micro-service/src/App.js

```diff
@@ -25,6 +26,9 @@ Sentry.init({
   tracesSampleRate: 1.0,
 });
 
+const appStoreUrl = process.env.NEXT_PUBLIC_APPLE_APP_STORE_LINK;
```

Comment:
These values should come from the database configs

Reply:
@redreceipt  do we config this from admin I cant find the config key of store url in database

Reply:
https://github.com/ApollosProject/apollos-cluster/blob/fa74dd419a0fc527832c7b3b38066f52ab9259f3/src/data/sharable/index.js#L30-L31